cmd/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/alecthomas/kong"
+	log "github.com/sirupsen/logrus"
+)
+
+//nolint:gochecknoglobals
+var cli struct {
+	Debug bool
+	Build Build `kong:"cmd,name='build',help='Build target',default='withargs'"`
+	List  List  `kong:"cmd,name='list',help='List available targets'"`
+	Run   Run   `kong:"cmd,name='run',help='Run executable target'"`
+}
+
+func getPrjRoot() (string, error) {
```

Comment:
I think it should look for `os.Getenv("PRJ_ROOT")` before falling back to calling git. That should make things faster and more portable to other SCM.